app/app.go

```diff
@@ -269,7 +270,7 @@ func New(
 			encodingConfig.TxConfig,
 		),
 		auth.NewAppModule(appCodec, app.AccountKeeper, nil),
-		// vesting.NewAppModule(app.AccountKeeper, app.BankKeeper),
+		vesting.NewAppModule(app.AccountKeeper, app.BankKeeper),
```

Comment:
Do we need to activate vesting module here? @tomtau

Reply:
otherwise, `chain-maind add-genesis-account --vesting` will fail, error is like unknown `cosmos.vesting.v1beta1.DelayedVestingAccount`.

Reply:
I think on master, it's possible to create vesting accounts after genesis, which we don't need, but I guess ok to activate?

Reply:
with new version, if not activate, we can't even add vesting account in genesis, I guess it need to register something.